index.html

```diff
@@ -286,8 +286,19 @@
           publisher: "ITU",
           date: "2021-04",
           href: "https://www.itu.int/ITU-T/recommendations/rec.aspx?rec=14652"
-        }
-      }
+        },  
+        "ITU-R BT.2408": {
+          title: "Guidance for operational practices in HDR television production",
+          publisher: "ITU",
+          date: "09/2023",
```

Comment:
I see that the [specref documentation says [rawDate is preferred](https://github.com/tobie/specref/blob/main/schemas/raw-reference.json#L153) and the format is YYYY-MM-DD with -DD or -MM-DD optional.